web/src/scheduled_messages.js

```diff
@@ -72,31 +73,29 @@ function hide_scheduled_message_success_compose_banner(scheduled_message_id) {
 }
 
 export function add_scheduled_messages(scheduled_messages) {
-    scheduled_messages_data.push(...scheduled_messages);
+    Object.assign(scheduled_messages_data, scheduled_messages);
     sort_scheduled_messages_data();
```

Comment:
Hmm, that seems incorrect since this what `scheduled_messages` looks like:
```
[
  {
    "scheduled_message_id": 17,
    "type": "private",
    "to": [6],
    "content": "Hello there!",
    "rendered_content": "<p>Hello there!</p>",
    "scheduled_delivery_timestamp": 1681662420,
    "failed": false,
  },
]
```

Reply:
Thanks for clarifying.
I have updated the code as per the following data. PTAL.